code/style.css

```diff
@@ -1,7 +1,356 @@
+  }
+  .question {
+    padding:0.5rem 1rem;
+    min-height: 3rem;
+    font-weight: bold;
+    display: flex;
+    align-items: center;
+    position: relative;
+    cursor: pointer;
+  }
+  .question::after {
+    content: "\25BE";
+    font-size: 1.5rem;
+    position: absolute;
+    right: 1rem;
+    transition: transform 0.2s ease-in-out;
+  }
+  .question.active::after {
+    /* content: "\2212";  */
+    transform: rotate(-180deg);
+  }
+  .answer {
+    max-height: 0;
+    overflow: hidden;
+    transition: max-height 0.2s ease-out;
+  }
+  
+  .answer p {
+    padding: 1rem;
+    background-color: whitesmoke;
+    border-top: 3px solid;
+    border-image: linear-gradient(to right, transparent, #5D463D,#EBD0C7) 1;
+  }
+  
+  /* FORM */
+  
+  input:valid {
+  /* What does that look like */
+  
+  }
+  input:invalid {
+  /*What does that look like*/
+  }
+  
+  
+  .form { 
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+  }
+  
+  .form img {
+    width: 10rem;
+    height: auto;
+    border-radius: 0.5rem;
+  }
+  
+  .login {
+    padding: 1rem;
+    width: 80%;
+    min-height: 650px;
+    max-width: 300px;
+    margin: 1rem;
+    border-radius: 0.5rem;
+    box-shadow: 0 4px 8px 0 rgba(0, 0, 0, 0.5);
+    border: 2px solid;
+    border-image: linear-gradient(to right, transparent, #5D463D,#EBD0C7) 1;
+    overflow: auto; 
+    display: flex;
+    flex-direction: column;
+    align-items: center; 
+    text-align: center;
+    
+  }
+  .signup {
+    padding: 1rem;
+    width: 80%;
+    max-height: 650px;
+    max-width: 300px;
+    margin: 1rem;
+    height: auto;
+    border-radius: 0.5rem;
+    box-shadow: 0 4px 8px 0 rgba(0, 0, 0, 0.5);
+    border: 2px solid;
+    border-image: linear-gradient(to right, transparent, #5D463D,#EBD0C7) 1;
+    text-align: center;
+  }
+  .help {
+    display: flex;
+    flex-direction: row;
+    width: 40%;
+  }
+  
+  select {
+    width: 100%;
+    padding: 10px 15px;
+    border: none;
+    border-radius: 4px;
+    background-color: whitesmoke;
+    color:#5D463D; 
+  }
+  input  {
+    width: 100%;
+    padding: 10px 15px;
+    margin: 4px 0;
+    border: 1px solid #ccc;
+    border-radius: 4px;
+    box-sizing: border-box;
+  }
+  
+  
+  button {
+    padding: 10px;
+    margin: 10px;
+    /* margin-left: 35%; */
+    color:white;
+    background-color: #90A8AD;
+    font-size: 20px;
+    font-family: 'Quattrocento', Serif;
+    border: 2px solid #90A8AD;
+    border-radius: 4%;
+  }
+  
+  button:hover {
+    transform: scale(1.1)
+    
+    }
+  
+  
+  
+  /*FOOTER*/
+  footer {
+    color: white;
+    padding: 20px;
+    height: 50px;
+    background-color:#90A8AD;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    flex-direction: column;
+    align-content: center;
+  }
+  }
+  
+    
+  
+  /* META */ 
+  /* if I delete this part the meta responsivness stops working, and I can't figure out why??*/ 
+  @media (min-width: 900px) {
```

Comment:
If you remove this whole media query and instead try to have min-width: 900px on the media query below, that should solve your issue 🎉